designs/administrator/olm/operand-form-yaml-switching/index.md

```diff
@@ -1,26 +1,38 @@
 ---
 parent: Administrator
```

Comment:
Curious if this should go to Design Conventions ... and then we can link back from our respective areas ?